<!DOCTYPE html>
<html>
<head>
  <title>Annotation Enrichment Analysis - An Alternative Method for Evaluating the Functional Properties of Gene Sets</title>
  <meta charset="utf-8">
  <meta name="description" content="Annotation Enrichment Analysis - An Alternative Method for Evaluating the Functional Properties of Gene Sets">
  <meta name="author" content="Keith Hughitt">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <hgroup class="auto-fadein">
    <h1>Annotation Enrichment Analysis - An Alternative Method for Evaluating the Functional Properties of Gene Sets</h1>
    <h2>Kimberly Glass, Michelle Girvan</h2>
    <p>Keith Hughitt<br/></p>
  </hgroup>
    <a href="https://github.com/khughitt/slidify-annotation_enrichment_analysis/zipball/gh-pages" class="example">
     Download
    </a>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <article data-timings="">
    <!-- Custom Styles -->

<style type='text/css'>
    .title-slide {
        background: #e3f2fc;
        text-shadow: 2px 2px 5px #666;
        color: #2d4152;
    }
    slides > slide {
        height: 800px;
        margin-top: -400px;
    }
    img {
        max-height: 560px;
        max-width: 964px;
    }
    slide a {border-bottom: none;}
    .small li { font-size: 20px; }
    .smaller li { font-size: 18px; }
    .caption { font-size: 14px; display: block; }
    .references li { font-size: 14px; }
    .references li p { font-size: 14px; }
</style>

<!-- Custom JavaScript -->

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>

<script type='text/javascript'>
$(function() {
    $("p:has(img)").addClass('centered');
});
</script>

<h2>Overview</h2>

<ul class = "build incremental">
<li>Recent high-throughput methods (microarray, RNA-Seq, etc) made it easy to
produce large datasets comparing samples in different conditions.</li>
<li>The end result of many of these analyses, however, is often a large list of
genes that are associated with one condition or the other.</li>
<li>Numerous tools have been developed to look for &quot;enrichment&quot; in these
resulting gene sets for genes associated with a particular known pathway or
functional annotation.</li>
<li>These methods (GSEA, etc) often use statistics which make some assumptions
about the distribution of annotations which may not be valid.</li>
<li>What are the effects of these assumptions the resulting interpretation?</li>
<li>Can we do better?</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-2" style="background:;">
  <hgroup>
    <h2>Example motivation: T. cruzi co-expression network</h2>
  </hgroup>
  <article data-timings="">
    <p>One example of where this kind of enrichment analysis could be useful is for
determining possible roles for clusters of co-expressed genes.</p>

<p><img src="assets/img/ModuleMembership.png" alt="tcruzi coex network"></p>

<p><span class='caption'>T. cruzi co-expression network modules detected by
WGCNA</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-3" style="background:;">
  <hgroup>
    <h2>Background</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="quote" id="slide-4" style="background:;">
  <hgroup>
    <h2>Gene Ontology (GO)</h2>
  </hgroup>
  <article data-timings="">
    <h3>What is GO?</h3>

<p><q>The Gene Ontology is a controlled vocabulary, a set of standard terms—words
and phrases—used for indexing and retrieving information. In addition to
defining terms, GO also defines the relationships between the terms, making
it a structured vocabulary.</q> - geneontology.org</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="small" id="slide-5" style="background:;">
  <hgroup>
    <h2>Gene Ontology (GO)</h2>
  </hgroup>
  <article data-timings="">
    <h3>What is GO?</h3>

<ul>
<li>Provides a common language to describe features of genes from all different
species.</li>
<li>GO database includes two main parts:

<ul>
<li>Ontologies</li>
<li>Gene annotations</li>
</ul></li>
<li>Includes three separate ontologies relating to:

<ul>
<li>Location (cellular component)</li>
<li>Process/pathway involved in (biological process)</li>
<li>Specific function (molecular function)</li>
</ul></li>
<li>Each ontology is represented by a directed acyclic graph (DAG).</li>
<li>Two different types of relationships exist between nodes: 

<ul>
<li><em>is-a</em>, and</li>
<li><em>part-of</em></li>
</ul></li>
<li>Deeper levels in the ontology correspond to more specific descriptions.</li>
<li>Maintained and developed by a consortium of scientists (<a href="http://www.geneontology.org/GO.consortiumlist.shtml">Gene Ontology
Consortium</a>)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>Gene Ontology (GO)</h2>
  </hgroup>
  <article data-timings="">
    <h3>Gene Ontology structure</h3>

<p><img src="assets/img/diag-ontology-graph.gif" alt="GO example">
<span class='caption'>
(source: <a href="http://www.geneontology.org/GO.ontology.structure.shtml">http://www.geneontology.org/GO.ontology.structure.shtml</a>)
</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>Many functional enrichment tools exist</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/Huang_table1.png" alt="Huang table 1">
<span class='caption'>Huang et al. (2009) Table 1</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-8" style="background:;">
  <hgroup>
    <h2>Common statistics used for enrichment analysis</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Fisher&#39;s Exact Test (FET)</li>
<li>Binomial test</li>
<li>Hypergeometric test</li>
<li>Chi-squared test</li>
</ul>

<p>All of these methods assume that, under the null hypothesis, genes are equally
likely to be selected.</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>Gene Set Enrichment Analysis (GSEA)</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Most popular tool for enrichment analysis</li>
<li>Uses variant of Kolmogorov–Smirnov test

<ul>
<li>Compares distributions of two samples</li>
<li>Null hypothesis: the samples were drawn from the same distribution</li>
</ul></li>
<li>Looks for enrichment in genes with a known property (e.g. GO annotation) at
the top of a list of genes ranked by differential expression, etc.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-10" style="background:;">
  <hgroup>
    <h2>Gene Set Enrichment Analysis (GSEA)</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/Subramanian_F1.large.jpg" alt="GSEA fig1">
<span class='caption'>Subramanian et al. (2005) Figure 1</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-11" style="background:;">
  <hgroup>
    <h2>GOstat</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Beißbarth &amp; Speed (2004)</li>
<li>Computes frequencies of all GO terms in two sets of genes:

<ul>
<li>Experiment set</li>
<li>Reference set (e.g. entire GO db)</li>
</ul></li>
<li>Uses \(\chi^2\) and Fisher&#39;s Exact test to look for terms which are enriched in
either gene set with respect to the other.</li>
<li>Performs multiple testing correction using either Holm or Benjamini and
Hochberg correction.</li>
<li>Website: <a href="http://gostat.wehi.edu.au/">http://gostat.wehi.edu.au/</a></li>
<li>Not to be confused with
&quot;<a href="http://www.bioconductor.org/packages/release/bioc/html/GOstats.html">GOstats</a>&quot;,
a Bioconductor package for working with GO and microarray data...</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-12" style="background:;">
  <hgroup>
    <h2>GOSeq</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Young et al. (2010) notice biases between gene length, number of reads, and 
differential expression.</li>
<li>They show that GO categories also show a length bias (many categories have
significantly longer or shorter genest than expected by chance), which
indicates that enrichment results could in turn be skewed by DE length bias.</li>
<li>GOSeq corrects for the length bias by random sampling of a fitted
distribution based on length and DE proportion.</li>
</ul>

<p><img src="assets/img/gb-2010-11-2-r14-2.jpg" alt="GOSeq fig2">
<span class='caption'>Young et al. (2010) Figure 2</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="smaller" id="slide-13" style="background:;">
  <hgroup>
    <h2>Fisher&#39;s Exact Test (FET)</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>The most common test statistic used for functional enrichment</li>
<li>Considers the overlap between experiment gene set and set of genes with some
known functional annotation.</li>
</ul>

<p><img src="assets/img/FET_example_wolfram.png" alt="Fisher&#39;s Exact test example">
<span class='caption'>(source:
<a href="http://mathworld.wolfram.com/FishersExactTest.html">http://mathworld.wolfram.com/FishersExactTest.html</a>)</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-14" style="background:;">
  <hgroup>
    <h2>Results</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-15" style="background:;">
  <hgroup>
    <h2>Gene ontology characteristics</h2>
  </hgroup>
  <article data-timings="">
    <h3>Gene-term graph</h3>

<ul>
<li>Downloaded all human gene-term associations from the
<a href="http://www.geneontology.org/">Gene Ontology website</a>.</li>
<li>Constructed a gene/annotation graph, represented by an 
\(n_G \times n_T\) adjacency matrix </li>
</ul>

<p><img src="assets/img/adj_matrix_small.png" alt="adj matrix"></p>

<ul>
<li>\(n_G\) - number of genes</li>
<li>\(n_T\) - number of GO terms</li>
<li>\(A_{ij} = 1\) - Gene \(i\) is annotated with term \(j\)</li>
<li>\(A_{ij} = 0\) - Gene \(i\) is not annotated with term \(j\)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-16" style="background:;">
  <hgroup>
    <h2>Gene ontology characteristics</h2>
  </hgroup>
  <article data-timings="">
    <h3>Gene and term degree distributions</h3>

<p><img src="assets/img/srep04191-f1.jpg" alt="fig 1"></p>

<ul>
<li><span class='red'>Biological Process</span> terms dominate the human
annotations.</li>
<li>Degree of term (\(k_t\)) distribution is &quot;heavy-tailed&quot;; most terms are
associated with only a few genes, but some terms are used for a huge 
number of genes.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-17" style="background:;">
  <hgroup>
    <h2>Gene ontology characteristics</h2>
  </hgroup>
  <article data-timings="">
    <h3>Biological Process</h3>

<p>The remainder of the results are based on the biological process ontology:</p>

<ul>
<li>656,783 annotations</li>
<li>15,213 genes (avg: 43.2 annotations)</li>
<li>10192 terms (avg: 64.4 annotations)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-18" style="background:;">
  <hgroup>
    <h2>Question: What is the effect of annotation database properties on functional enrichment analysis?</h2>
  </hgroup>
  <article data-timings="">
    <h3>Experiment design:</h3>

<ul>
<li>Created 200 random gene sets:

<ul>
<li>\(N_g\)=200 genes in each set (a &quot;typical&quot; gene set size)</li>
<li>Varied number of annotations (\(M_g\))</li>
<li>Determined FET enrichment score for each of the 10192 BP GO terms</li>
</ul></li>
</ul>

<h3>Results</h3>

<ul>
<li>Number of unique annotations \(\propto\) GO enrichment significance!</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-19" style="background:;">
  <hgroup>
    <h2>Random gene set enrichment scores</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/srep04191-f2a-b.jpg" alt="fig2a-b"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-20" style="background:;">
  <hgroup>
    <h2>Perhaps the problem isn&#39;t quite so bad after correcting for multiple testing...</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Multiple testing correction alone is not enough to deal with this bias,
although it does seem to severely reduce the problem.</li>
</ul>

<p><img src="assets/img/srep04191-fs2.jpg" alt="figs2"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-21" style="background:;">
  <hgroup>
    <h2>Annotation Enrichment Analysis (AEA)</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/srep04191-f3.jpg" alt="fig3"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-22" style="background:;">
  <hgroup>
    <h2>Question: Does this bias also affect biologically relevant sets of genes?</h2>
  </hgroup>
  <article data-timings="">
    <h3>Overview</h3>

<ul>
<li>So far, we&#39;ve seen how AEA can correct for biases in the distribution of GO
terms and annotation coverage across genes.</li>
<li>Does this have any impact on downstream biological interpretations?</li>
</ul>

<h3>Experiment design:</h3>

<ul>
<li>Downloaded all expression signatures from
<a href="http://compbio.dfci.harvard.edu/genesigdb/">Gene Signatures Database (GeneSigDB)</a> which contain \(100 <=
n <= 1000\) genes which are annotated with a term in the BP ontology
(total=309)</li>
<li>First, plotted average number of annotations per gene set and compared it what would
be expected for random sets of genes (verify presence of bias.)</li>
<li>Next, measured enrichment in each set using FET and AEA and looked at
properties of genes deemed significant in one measure but not the other.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-23" style="background:;">
  <hgroup>
    <h2>GeneSigDB Signatures</h2>
  </hgroup>
  <article data-timings="">
    <p>FET enrichment bias towards well-annotated genes is also present in biological
datasets:</p>

<p><img src="assets/img/srep04191-f4.jpg" alt="fig4"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-24" style="background:;">
  <hgroup>
    <h2>Question: Does AEA provide any additional biological insights?</h2>
  </hgroup>
  <article data-timings="">
    <h3>Experiment design:</h3>

<ul>
<li>For FET and AEA, each:

<ul>
<li>Selected ~40 <span class='blue2'>GeneSigDB signatures</span> with the 
most significant enrichment scores.</li>
<li>Select 40 <span class='blue2'>GO terms</span> with most significant 
enrichment scores across all signatures.</li>
</ul></li>
<li>Performed hierarchical clustering.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-25" style="background:;">
  <hgroup>
    <h2>Functional enrichment clusters (FET vs. AEA)</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/srep04191-f5.jpg" alt="fig5"></p>

<ul>
<li>rows = gene signatures</li>
<li>columns = GO terms</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-26" style="background:;">
  <hgroup>
    <h2>Real andi random term-signature comparisons</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/srep04191-f6.jpg" alt="fig6"></p>

<ul>
<li>Created random term sets with same number of unique genes annotated as found
in real GO branches</li>
<li>Measured FET/AEA enrichment in random and real go branches</li>
<li>FET find similar numbers of &quot;significant&quot; term-signature associations in the
real and random branches!</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-27" style="background:;">
  <hgroup>
    <h2>Conclusions</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-28" style="background:;">
  <hgroup>
    <h2>Conclusions</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Biases exist in GO and other annotation databases.</li>
<li>These biases can affect the performance of statistics such as FET in
predicting significant enrichment.</li>
<li>Annotation Enrichment Analysis (AEA) accounts for these biases and is able is
not as prone to detecting spurious enrichments.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-29" style="background:;">
  <hgroup>
    <h2>Limitations</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Performance of AEA only compared with Fisher&#39;s Exact Test (FET); how does the
performance compare to other GO methods?</li>
<li>Only looked at Biological Process ontology -- is the picture the same for the
other GO sub-ontologies? Other annotation databases?</li>
<li>Currently only implemented in C++ (R bindings would be nice.)</li>
<li>Does not take into account any of the addition information about the members
of the experimental gene set (e.g. DE fold-change, p-value, etc)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-30" style="background:;">
  <hgroup>
    <h2>Beyond the Gene Ontology...</h2>
  </hgroup>
  <article data-timings="">
    <iframe src="http://amp.pharm.mssm.edu/Enrichr/"></iframe>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="references" id="slide-31" style="background:;">
  <hgroup>
    <h2>References</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>T. Beissbarth, T. P. Speed,   (2004) Gostat: Find Statistically Overrepresented Gene Ontologies Within A Group of Genes.  <em>Bioinformatics</em>  <strong>20</strong>  1464-1465  <a href="http://dx.doi.org/10.1093/bioinformatics/bth088">10.1093/bioinformatics/bth088</a></li>
<li>Kimberly Glass, Michelle Girvan,   (2014) Annotation Enrichment Analysis: an Alternative Method For Evaluating The Functional Properties of Gene Sets.  <em>Scientific Reports</em>  <strong>4</strong>  <a href="http://dx.doi.org/10.1038/srep04191">10.1038/srep04191</a></li>
<li>D. W. Huang, B. T. Sherman, R. A. Lempicki,   (2008) Bioinformatics Enrichment Tools: Paths Toward The Comprehensive Functional Analysis of Large Gene Lists.  <em>Nucleic Acids Research</em>  <strong>37</strong>  1-13  <a href="http://dx.doi.org/10.1093/nar/gkn923">10.1093/nar/gkn923</a></li>
<li>A. Subramanian, P. Tamayo, V. K. Mootha, S. Mukherjee, B. L. Ebert, M. A. Gillette, A. Paulovich, S. L. Pomeroy, T. R. Golub, E. S. Lander, J. P. Mesirov,   (2005) Gene Set Enrichment Analysis: A Knowledge-Based Approach For Interpreting Genome-Wide Expression Profiles.  <em>Proceedings of The National Academy of Sciences</em>  <strong>102</strong>  15545-15550  <a href="http://dx.doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a></li>
<li><p>Matthew D Young, Matthew J Wakefield, Gordon K Smyth, Alicia Oshlack,   (2010) Gene Ontology Analysis For Rna-Seq: Accounting For Selection Bias.  <em>Genome Biology</em>  <strong>11</strong>  R14-NA  <a href="http://dx.doi.org/10.1186/gb-2010-11-2-r14">10.1186/gb-2010-11-2-r14</a></p></li>
<li><p>Weisstein, Eric W. &quot;Fisher&#39;s Exact Test.&quot; From MathWorld--A Wolfram Web
Resource. <a href="http://mathworld.wolfram.com/FishersExactTest.html">http://mathworld.wolfram.com/FishersExactTest.html</a></p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title=''>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Example motivation: T. cruzi co-expression network'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Background'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Gene Ontology (GO)'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Gene Ontology (GO)'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Gene Ontology (GO)'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='Many functional enrichment tools exist'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='Common statistics used for enrichment analysis'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='Gene Set Enrichment Analysis (GSEA)'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='Gene Set Enrichment Analysis (GSEA)'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='GOstat'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='GOSeq'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='Fisher&#39;s Exact Test (FET)'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='Results'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='Gene ontology characteristics'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='Gene ontology characteristics'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='Gene ontology characteristics'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='Question: What is the effect of annotation database properties on functional enrichment analysis?'>
         18
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=19 title='Random gene set enrichment scores'>
         19
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=20 title='Perhaps the problem isn&#39;t quite so bad after correcting for multiple testing...'>
         20
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=21 title='Annotation Enrichment Analysis (AEA)'>
         21
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=22 title='Question: Does this bias also affect biologically relevant sets of genes?'>
         22
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=23 title='GeneSigDB Signatures'>
         23
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=24 title='Question: Does AEA provide any additional biological insights?'>
         24
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=25 title='Functional enrichment clusters (FET vs. AEA)'>
         25
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=26 title='Real andi random term-signature comparisons'>
         26
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=27 title='Conclusions'>
         27
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=28 title='Conclusions'>
         28
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=29 title='Limitations'>
         29
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=30 title='Beyond the Gene Ontology...'>
         30
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=31 title='References'>
         31
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>