<!DOCTYPE html>
<html>
<head>
  <title>Annotation Enrichment Analysis - An Alternative Method for Evaluating the Functional Properties of Gene Sets</title>
  <meta charset="utf-8">
  <meta name="description" content="Annotation Enrichment Analysis - An Alternative Method for Evaluating the Functional Properties of Gene Sets">
  <meta name="author" content="Keith Hughitt">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <hgroup class="auto-fadein">
    <h1>Annotation Enrichment Analysis - An Alternative Method for Evaluating the Functional Properties of Gene Sets</h1>
    <h2>Kimberly Glass, Michelle Girvan</h2>
    <p>Keith Hughitt<br/></p>
  </hgroup>
    <a href="https://github.com/khughitt/slidify-annotation_enrichment_analysis/zipball/gh-pages" class="example">
     Download
    </a>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <article data-timings="">
    <!-- Custom Styles -->

<style type='text/css'>
    .title-slide {
        background: #e3f2fc;
        color: #2d4152;
    }
    slides > slide {
        height: 800px;
        margin-top: -400px;
    }
    img {
        max-height: 560px;
        max-width: 964px;
    }
    slide a {border-bottom: none;}
    .small li { font-size: 20px; }
    .smaller li p { font-size: 16px; }
    .caption { font-size: 14px; display: block; }
    .references li { font-size: 14px; }
    .references li p { font-size: 14px; }
</style>

<!-- Custom JavaScript -->

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>

<script type='text/javascript'>
$(function() {
    $("p:has(img)").addClass('centered');
});
</script>

<h2>Overview</h2>

<ul class = "build incremental">
<li>Recent high-throughput methods (microarray, RNA-Seq, etc) made it easy to
produce large datasets comparing samples in different conditions.</li>
<li>The end result of many of these analyses, however, is often a large list of
genes that are associated with one condition or the other.</li>
<li>Numerous tools have been developed to look for &quot;enrichment&quot; in these
resulting gene sets for genes associated with a particular known pathway or
functional annotation.</li>
<li>These methods (GSEA, etc) often use statistics which make some assumptions
about the distribution of annotations which may not be valid.</li>
<li>What are the effects of these assumptions the resulting interpretation?</li>
<li>Can we do better?</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-2" style="background:;">
  <hgroup>
    <h2>Background</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-3" style="background:;">
  <hgroup>
    <h2>Many functional enrichment tools exist</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/Huang_table1.png" alt="Huang table 1">
<span class='caption'>Huang et al. (2009) Table 1</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-4" style="background:;">
  <hgroup>
    <h2>Common statistics used for enrichment analysis</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Fisher&#39;s Exact Test (FET)</li>
<li>Binomial test</li>
<li>Hypergeometric test</li>
<li>Chi-squared test</li>
</ul>

<p>All of these methods assume that, under the null hypothesis, genes are equally
likely to be selected.</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-5" style="background:;">
  <hgroup>
    <h2>Gene Set Enrichment Analysis (GSEA)</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Most popular tool for enrichment analysis</li>
<li>Uses variant of Kolmogorovâ€“Smirnov test

<ul>
<li>Compares distributions of two samples</li>
<li>Null hypothesis: the samples were drawn from the same distribution</li>
</ul></li>
<li>Looks for enrichment in genes with a known property (e.g. GO annotation) at
the top of a list of genes ranked by differential expression, etc.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>Gene Set Enrichment Analysis (GSEA)</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/Subramanian_F1.large.jpg" alt="GSEA fig1">
<span class='caption'>Subramanian et al. (2005) Figure 1</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>Fisher&#39;s Exact Test (FET)</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>The most common test statistic used for functional enrichment</li>
<li>Considers the overlap between experiment gene set and set of genes with some
known functional annotation.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-8" style="background:;">
  <hgroup>
    <h2>Annotation Enrichment Analysis</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>Annotation Enrichment Analysis</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Downloaded all human gene-term associations from
<a href="http://www.geneontology.org/">GO</a>.</li>
<li>Constructed a gene/annotation bipartite graph, represented by an 
\(n_G \cross n_T\) adjacency matrix </li>
</ul>

<p><img src="assets/img/adj_matrix.png" alt="adj matrix"></p>

<ul>
<li>\(n_G\) - number of genes</li>
<li>\(n_T\) - number of GO terms</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="references" id="slide-10" style="background:;">
  <hgroup>
    <h2>References</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Kimberly Glass, Michelle Girvan,   (2014) Annotation Enrichment Analysis: an Alternative Method For Evaluating The Functional Properties of Gene Sets.  <em>Scientific Reports</em>  <strong>4</strong>  <a href="http://dx.doi.org/10.1038/srep04191">10.1038/srep04191</a></li>
<li>D. W. Huang, B. T. Sherman, R. A. Lempicki,   (2008) Bioinformatics Enrichment Tools: Paths Toward The Comprehensive Functional Analysis of Large Gene Lists.  <em>Nucleic Acids Research</em>  <strong>37</strong>  1-13  <a href="http://dx.doi.org/10.1093/nar/gkn923">10.1093/nar/gkn923</a></li>
<li>A. Subramanian, P. Tamayo, V. K. Mootha, S. Mukherjee, B. L. Ebert, M. A. Gillette, A. Paulovich, S. L. Pomeroy, T. R. Golub, E. S. Lander, J. P. Mesirov,   (2005) Gene Set Enrichment Analysis: A Knowledge-Based Approach For Interpreting Genome-Wide Expression Profiles.  <em>Proceedings of The National Academy of Sciences</em>  <strong>102</strong>  15545-15550  <a href="http://dx.doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title=''>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Background'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Many functional enrichment tools exist'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Common statistics used for enrichment analysis'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Gene Set Enrichment Analysis (GSEA)'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Gene Set Enrichment Analysis (GSEA)'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='Fisher&#39;s Exact Test (FET)'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='Annotation Enrichment Analysis'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='Annotation Enrichment Analysis'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='References'>
         10
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>